clerk/sessions_test.go

```diff
@@ -242,6 +243,47 @@ func TestSessionsService_Verify_invalidServer(t *testing.T) {
 	}
 }
 
+func TestSessionsService_CreateTokenFromTemplate_Success(t *testing.T) {
+	token := "token"
+	sessionId := fmt.Sprintf("sess_%s", "2bp7pawkvFUR3m3QPBm6Cwx5ghZ")
```

Comment:
This `Sprintf` call is probably not needed. There's no variable substitution here. Same for line 278 below.